config/test.exs

```diff
@@ -34,7 +34,7 @@ config :lightning, LightningWeb.Endpoint,
   url: [host: "localhost", port: 4002],
   secret_key_base:
     "/8zedVJLxvmGGFoRExE3e870g7CGZZQ1Vq11A5MbQGPKOpK57MahVsPW6Wkkv61n",
-  server: false
+  server: true
```

Comment:
I'm a little wary of enabling the server globally for testing.

This means that we would have the worker creating records without being in a test transaction right? We'd have to clean up before and after running the tests and they would have to be synchronous so we don't introduce flakiness.

What would be first prize is we start the server inside the test, and leverage the Ecto sandbox behaviour.

Reply:
It's okay Stu, everything happens related to a workorder that is **not persisted** even created by a webhook with `server: true` and the sandbox tests are not affected by persisted records.